src/flexbox.rs

```diff
@@ -95,7 +95,7 @@ struct AlgoConstants {
     padding_border: Rect<f32>,
 
     /// The size of the internal node
-    node_inner_size: Size<Option<f32>>,
+    node_inner_size: Size<f32>,
     /// The size of the surrounding container
     container_size: Size<f32>,
```

Comment:
Looking at this, do we need a `Size` that does not have Optional fields? One that is guaranteed to have both width and height?